.github/workflows/build-test.yml

```diff
@@ -55,6 +55,7 @@ jobs:
       - name: Test
         run: CGO_ENABLED=1 go test -tags=testutils -race -p 1 -v ./...
         env:
+          ZEP_LOG_LEVEL: 'info'
           ZEP_OPENAI_API_KEY: ${{ secrets.ZEP_OPENAI_API_KEY }}
           ZEP_ANTHROPIC_API_KEY: ${{ secrets.ZEP_ANTHROPIC_API_KEY }}
           ZEP_STORE_POSTGRES_DSN: 'postgres://postgres:postgres@postgres:5432/?sslmode=disable'
```

Comment:
Obviously db credentials here are exposed. It looks like it's local to this machine (docker). I'm not sure that I care so much but thought it was worth pointing out

Reply:
Yep - those are default credentials. No problem exposing them.